mmpose/datasets/datasets/top_down/topdown_carfusion_dataset.py

```diff
@@ -0,0 +1,403 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os.path as osp
+import tempfile
+import warnings
+from collections import OrderedDict, defaultdict
+
+import json_tricks as json
+import numpy as np
+from mmcv import Config, deprecated_api_warning
+from xtcocotools.cocoeval import COCOeval
+
+from ....core.post_processing import oks_nms, soft_oks_nms
+from ...builder import DATASETS
+from ..base import Kpt2dSviewRgbImgTopDownDataset
+
+
+@DATASETS.register_module()
+class TopDownCarFusionDataset(Kpt2dSviewRgbImgTopDownDataset):
+    """CocoDataset dataset for top-down pose estimation.
+
+    The dataset loads raw features and apply specified transforms
+    to return a dict containing the image tensors and other information.
+
+    CarFusion keypoint indexes::
+        0: 'Right Front wheel',
+        1: 'Left Front wheel'
+        2: 'Right Back wheel'
+        3: 'Left Back wheel'
+        4: 'Right Front HeadLight'
+        5: 'Left Front HeadLight'
+        6: 'Right Back HeadLight'
+        7: 'Left Back HeadLight'
+        8: 'Exhaust'
+        9:
```

Comment:
Why this line is left blank?

Reply:
9: keypoint is miscellaneous in the dataset